File: 1 - BEGINNER/Java/1019 - Conversão de Tempo.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
 
        Scanner input = new Scanner(System.in);
        
        int N = input.nextInt();
        int horas = N / 3600;
        N = N % 3600;
        int minutos = N / 60;
        N = N % 60;
        
        System.out.printf("%d:%d:%d\n", horas, minutos, N);
        
        input.close();
        
    }
 
}

File: 1 - BEGINNER/C++/1019 - Conversão de Tempo.cpp
#include <iostream>

using namespace std;
 
int main() {
 
    int N, segundos, minutos, horas;
    
    cin >> N;
    
    horas = N / 3600;
    N %= 3600;
    minutos = N / 60;
    N %= 60;
    
    cout << horas << ":" << minutos << ":" << N << endl;
 
    return 0;
}